src/tests/test_openclpow.py

```diff
@@ -0,0 +1,28 @@
+"""
+Tests for openclpow module
+"""
+import hashlib
+import unittest
+
+import pybitmessage.openclpow as openclpow
+
+from struct import pack, unpack
+
+
+class TestOpenClPow(unittest.TestCase):
+    """
+    Main opencl test case
+    """
+
+    @unittest.skipIf(not openclpow.libAvailable,
```

Comment:
@unittest.skipIf(not openclpow.enabledGpus,
                                "No GPUs found / enabled")